### PR TITLE
ENT-1236 Add sorting for /users endpoint

### PR DESCRIPTION
**Pull request by irfanuddinahmad**
Sorting for /users endpoint in edx-enterprise-data not configured
https://openedx.atlassian.net/browse/ENT-1236

### CODE REVIEW

File: enterprise_data/models.py [hunk 1]
```diff
@@ -97,7 +97,6 @@ class Meta:
         db_table = 'enterprise_user'
         verbose_name = _("Enterprise User")
         verbose_name_plural = _("Enterprise Users")
-        ordering = ['-user_email']
```
Comment (muhammad-ammar) [inline enterprise_data/models.py]: @irfanuddinahmad I think we don't need to remove this.

File: enterprise_data/api/v0/views.py [hunk 2]
```diff
@@ -241,6 +244,9 @@ def list(self, request, **kwargs):
         elif all_enrollments_passed == 'false':
             users = users.filter(enrollments__has_passed=False)
 
+        # do the sorting
+        users = self.filter_queryset(users)
```
Comment (muhammad-ammar) [inline enterprise_data/api/v0/views.py:248]: @irfanuddinahmad This is what we needed for ordering.

File: enterprise_data/tests/test_views.py [hunk 3]
```diff
@@ -566,6 +566,38 @@ def test_no_page_querystring_skips_pagination(self):
...
+        url = reverse(
+            'v0:enterprise-users-list',
+            kwargs=kwargs,
+        )
+        response = self.client.get(url, params)
+        assert response.json()['count'] == 4
+        assert response.json()['results'][0]['id'] == self.ent_user4.id
+        assert response.json()['results'][3]['id'] == self.ent_user8.id
+
+    def test_viewset_ordering_reverse(self):
```
Comment (zubair-arbi) [inline enterprise_data/tests/test_views.py]: Please use ddt for combining both tests.

**Review by zubair-arbi — COMMENTED** [review]
Just a suggestion about combining two tests, otherwise changes looks good.

**Review by zubair-arbi — APPROVED** [review]
LGTM 👍

**Review by mushtaqak — APPROVED** [review]
Looks good 👍

**Comment by mushtaqak** [comment]
@irfanuddinahmad Seems like `EnterpriseLearnerCompletedCoursesViewSet` too does not have sorting enabled. I am verifying and will add a commit adding test for it, if need fix.

**Comment by mushtaqak** [comment]
@irfanuddinahmad You can merge your changes. I have created another PR https://github.com/edx/edx-enterprise-data/pull/71 that will address the concerned mentioned above.

File: enterprise_data/api/v0/views.py [hunk 4]
```diff
@@ -213,6 +213,9 @@ class EnterpriseUsersViewSet(EnterpriseViewSet, viewsets.ModelViewSet):
     """
     queryset = EnterpriseUser.objects.all()
     serializer_class = serializers.EnterpriseUserSerializer
+    filter_backends = (filters.OrderingFilter,)
+    ordering_fields = '__all__'
```
Comment (mushtaqak) [inline enterprise_data/api/v0/views.py:217]: and yeah we need to add changes to `CHANGELOG` as well.

**Pull request merged**